js/app.js

```diff
@@ -0,0 +1,147 @@
+var object;
```

Comment:
where is the IEFE, please ask other's if you have not understood a concept, better early than late